helfi_platform_config.module

```diff
@@ -561,6 +570,24 @@ function helfi_platform_config_form_alter(&$form, FormStateInterface $form_state
   }
 }
 
+/**
+ * Implements hook_form_BASE_FORM_ID_alter() for "search_api_index_form".
+ */
+function helfi_platform_config_form_search_api_index_form_alter(&$form, FormStateInterface $form_state, $form_id): void {
+  $formObject = $form_state->getFormObject();
+  assert($formObject instanceof EntityFormInterface);
+  $index = $formObject->getEntity();
+  assert($index instanceof IndexInterface);
+
+  // Add an index option to enable site specific item id prefix.
+  $form['options']['helfi_platform_config_multisite'] = [
+    '#type' => 'checkbox',
+    '#title' => new TranslatableMarkup('Multisite index'),
+    '#description' => new TranslatableMarkup('Enabling this option will add a site specific prefix to all item ids, based on environment resolver site name, to avoid collapsing id values.'),
+    '#default_value' => $index->getOption('helfi_platform_config_multisite'),
+  ];
+}
```

Comment:
Just to make this even simpler, I don't think we necessarily need to add UI for this feature. I think it is perfectly fine to enable to multi-site feature directly on the config files. I don't think we need to do that too often. However, I'm not opposed to this if you think this is nice to have.